### PR TITLE
Include config script to tinker params in upkeep manager and cancel members

### PR DESCRIPTION
**Pull request by petrovska-petro**
Tackles #1152 

```
brownie run upkeep_manager/set_config  cancel_member

brownie run upkeep_manager/set_config  set_min_rounds_topup 5

brownie run upkeep_manager/set_config  set_rounds_topup 10

brownie run upkeep_manager/set_config  pause

brownie run upkeep_manager/set_config  unpause

### CODE REVIEW

**Review by gosuto-inzasheru — COMMENTED** [review]
better to not have default args to avoid mistakes

File: scripts/upkeep_manager/set_config.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+from helpers.addresses import r
+
+
+techops = GreatApeSafe(r.badger_wallets.techops_multisig)
+
+# contract
+upkeep_manager = techops.contract(r.badger_wallets.upkeep_manager)
+
+
+def cancel_member(targert_member=r.safe_modules.treasury_voter.aura_auto_lock):
```
[the remark this answers is not in the document]
Comment (petrovska-petro) [reply scripts/upkeep_manager/set_config.py]: removed in 7c0105f61b1afdded0d280edbb30bd87561e468e

left initially a default arg to facilitate posting later, but its true that for future rounds could lead to a mistake

File: scripts/upkeep_manager/set_config.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    """
+    members = upkeep_manager.getMembers()
+    assert targert_member in members
+
+    upkeep_manager.cancelMemberUpkeep(targert_member)
+
+    techops.post_safe_tx()
+
+
+def set_min_rounds_topup(rounds=0):
```
[the remark this answers is not in the document]
Comment (petrovska-petro) [reply scripts/upkeep_manager/set_config.py]: 7c0105f61b1afdded0d280edbb30bd87561e468e

File: scripts/upkeep_manager/set_config.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    """
+    rounds = int(rounds)
+    assert upkeep_manager.minRoundsTopUp() != rounds
+
+    upkeep_manager.setMinRoundsTopUp(rounds)
+
+    techops.post_safe_tx()
+
+
+def set_rounds_topup(rounds=0):
```
[the remark this answers is not in the document]
Comment (petrovska-petro) [reply scripts/upkeep_manager/set_config.py]: 7c0105f61b1afdded0d280edbb30bd87561e468e

**Review by jalbrekt85 — APPROVED** [review]
useful functions for interacting with upkeep manager, lgtm

File: scripts/upkeep_manager/set_config.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (gosuto-inzasheru) [reply scripts/upkeep_manager/set_config.py]: `=None` is still a default arg 😛

**Review by gosuto-inzasheru — APPROVED** [review]

**Pull request merged**